pandas/_libs/lib.pyx

```diff
@@ -841,19 +845,19 @@ def indices_fast(object index, ndarray[int64_t] labels, list keys,
 
 # core.common import for fast inference checks
 
-def is_float(obj: object) -> bint:
+def is_float(obj: object) -> bool:
     return util.is_float_object(obj)
 
 
-def is_integer(obj: object) -> bint:
+def is_integer(obj: object) -> bool:
     return util.is_integer_object(obj)
 
 
-def is_bool(obj: object) -> bint:
+def is_bool(obj: object) -> bool:
```

Comment:
why are these changing? what is wrong with bint?

Reply:
nothing is wrong with bint, but part of the upside of using py3-annotations is that its valid python.  using bint instead of bool defeats that purpose